src/adv.py

```diff
@@ -38,29 +38,26 @@
 # Main
 #
 
-# Make a new player object that is currently in the 'outside' room.
-player_name = input('Welcome, adventurer!  What is your name?\n')
-player = Player(player_name, 'outside')
+# Greeting:
+player_name = input('\nWelcome, adventurer!  What is your name?\n')
+
+# Start position:
+player = Player(player_name, room['outside'])
+
+print(f'\nYou find yourself next to an {player.current_room.name}\n{player.current_room.description}')
 
-print(f'\n{player.name} is standing {player.current_room} ')
+valid_dir = ['n', 's', 'e', 'w']
 
-cmds = input('\nType "c" to continue. \nType "q" to quit the game.\n')
+cmds = input('\nChoose a direction to move:\n   "n" - North\n   "s" - South\n   "e" - East\n   "w" - West\n Type "q" to quit the game.\n')
 
-# Write a loop that:
 while not cmds == 'q':
 
-    # * Prints the current room name
-    if cmds == 'c':
-        print(room[player.current_room].name)
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
-#
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
-#
-# If the user enters "q", quit the game.
+    if valid_dir.__contains__(cmds):
```

Comment:
I believe this could also be done while avoiding the `__contains__` dunder method just by changing the format to `if cmds in valid_dir` with the `in` keyword doing the check. Give it a try!